### PR TITLE
Add host genome

### PR DESCRIPTION
**Pull request by yunfangjuan**
add host genome to the samples page and prepare the wiring for default report generation.

### CODE REVIEW

File: db/migrate/20171011225553_create_host_genomes.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class CreateHostGenomes < ActiveRecord::Migration[5.1]
+  def change
+    create_table :host_genomes do |t|
+      t.string :name, null:false, unique: true
+      t.text :s3_star_index_path
+      t.text :s3_bowtie2_index_path
+      t.bigint :default_background_id
```
Comment (boris-dimitrov) [inline db/migrate/20171011225553_create_host_genomes.rb]: is this a circular dependency?

  -- in order to create background, you need several runs

  -- for each run, you need a host genome

  -- for a host genome, you need a background?

File: lib/tasks/host_genomes.rake [hunk 2]
```diff
@@ -0,0 +1,10 @@
+require 'open3'
```
Comment (boris-dimitrov) [inline lib/tasks/host_genomes.rake]: what runs this task?

File: db/migrate/20171011225553_create_host_genomes.rb (hunk 1 above)
Comment (yunfangjuan) [reply db/migrate/20171011225553_create_host_genomes.rb]: well that's why it's called default_background_id. 

For each host genome, there would be a background that consists of runs that are run based on the host genome, which has the indexes that areapecified in the beginning. 

Once those are done, a default could be created but it's not there in the beginning.

File: lib/tasks/host_genomes.rake (hunk 2 above)
Comment (yunfangjuan) [reply lib/tasks/host_genomes.rake]: guess we don't need it here.

**Comment by yunfangjuan** [comment]
Run 

docker-compose run web "rake create_host_genomes" 

To initialize host genomes, (human/mosquito)

**Pull request merged**